ccdproc/image_collection.py

```diff
@@ -216,6 +223,13 @@ def files(self):
         """
         return self._files
 
+    @property
+    def ext(self):
+        """
+        str,The extension for which all 
```

Comment:
I don't think I accounted for that. The requirement in the original issue was that a single extension would be extracted from all files.
So I don't know if any iterables like a tuple would work.